resources/themes/cura-light/icons/low/Warning.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M13 6H11V13H13V6ZM13 15H11V17H13V15Z" fill="#000E1A"/>
```

Comment:
The `clip-rule` property is not supported by Qt.
The `fill-rule` property is not used in this image since there are no self-intersections.
The `fill` property is not used in this image, since it is always recoloured. The colour shown here is incorrect (different from what we actually use in the interface).